docs/user/docker.md

```diff
@@ -30,91 +45,100 @@ To save the experiment data generated in the container, you need to specify a
 path where the files will be saved inside your host computer with the argument
 `-v` in the docker-run command. For example, if the path you want to use is
 at `/home/tmp/data`, execute:
+
 ```
 docker run \
   -it \
   --rm \
-  -v /home/tmp/data:/root/code/garage/data \
+  -v /home/tmp/data:/home/garage-user/code/garage/data \
   rlworkgroup/garage-headless python examples/tf/trpo_cartpole.py
 ```
+
 This binds a volume between your host path and the path in garage at
-`/root/code/garage/data`.
+`/home/garage-user/code/garage/data`. Make sure the directory at the host
+path exists, otherwise docker won't be able to write to it.
 
-### Build and run the image
+### Build and run the headless image
 
 To build the headless image, first clone this repository, move to the root
 folder of your local repository and then execute:
+
 ```
 make build-headless
 ```
 
 To build and run the container, execute;
+
 ```
 make run-headless RUN_CMD="python examples/tf/trpo_cartpole.py"
 ```
+
 Where RUN_CMD specifies the executable to run in the container.
 
 The previous command adds a volume from the data folder inside your cloned
 garage repository to the data folder in the garage container, so any experiment
 results ran in the container will be saved in the data folder inside your
-cloned repository. The data is saved in a folder with the name of the container
-that generated the data, which by default is the name of the image type the
-container is based on with the date and time the container was launched.
+cloned repository. The data is saved in a folder named `data` inside the current
+working directory. You can override that by using the variable `DATA_PATH` and
+passing the *absolute path* to the output directory.
 
-If you want to specify another name for the container, do so with the variable
-`CONTAINER_NAME`:
 ```
-make run-headless RUN_CMD="..." CONTAINER_NAME="my_container"
+make run-headless RUN_CMD="..." DATA_PATH="/home/xyz/my_garage_exp"
+```
+
+By default, docker generates random names for containers. If you want to specify
+a name for the container, do so with the variable `CONTAINER_NAME`.
+
+```
+make run-headless RUN_CMD="..." CONTAINER_NAME="my_container_123"
 ```
 
 If you need to use MuJoCo, you need to place your key at `~/.mujoco/mjkey.txt`
 or specify the corresponding path through the MJKEY_PATH variable:
+
 ```
 make run-headless RUN_CMD="..." MJKEY_PATH="/home/user/mjkey.txt"
 ```
 
-If you require to pass addtional arguments to the the make commands, you can
-use the variable ADD_ARGS, for example:
+If you require to pass additional arguments to docker build and run commands,
+you can use the variables BUILD_ARGS and RUN_ARGS, for example:
+
 ```
-make build-headless ADD_ARGS="--build-arg MY_VAR=123"
-make run-headless ADD_ARGS="-e MY_VAR=123"
+make build-headless BUILD_ARGS="--build-arg MY_VAR=123"
+make run-headless RUN_ARGS="-e MY_VAR=123"
 ```
 
-#### Prerequisites
+## nvidia image
```

Comment:
It reads kinda weird that this subsection isn't capitalized.

Reply:
Capitalized now